Sources/_StringProcessing/RegexDSL/Core.swift

```diff
@@ -47,6 +47,14 @@ public struct Regex<Capture>: RegexProtocol {
     self.init(ast: try! parse(pattern, .traditional))
   }
 
+  // Compiler interface. Do not change independently.
+  @usableFromInline
+  init(_regexString pattern: String, version: Int) {
+    // The version argument is passed by the compiler using the value defined
+    // in libswiftParseRegexLiteral.
```

Comment:
Can we assert its equal to 1, at least for now? And/or make an internal global for current version number, and we'll expand on that if/when we need a new one

Reply:
How's this?